app/models.py

```diff
@@ -1339,8 +1339,10 @@ def subject(self):
         from app.utils import get_template_instance
 
         if self.notification_type != SMS_TYPE:
```

Comment:
Do push notifications have a subject?  This might need to test `== EMAIL_TYPE`.

Reply:
push notifications do not use any of our templates (yet).